src/components/auth/EthWalletSelector.tsx

```diff
@@ -68,8 +69,9 @@ export const EthWalletSelector: React.FC<WalletSelectorProps> = React.memo(({
       </div>}
       {loading && <Loader active={loading} provider={provider} size="massive" />}
       {!isElectron() && <LoginModal.Option type={browserWallet || LoginModalOptionType.METAMASK} onClick={handleLoginInjected} />}
-      <LoginModal.Option type={LoginModalOptionType.FORTMATIC} onClick={handleLoginFortmatic} />
+      {!isElectron() && <LoginModal.Option type={LoginModalOptionType.FORTMATIC} onClick={handleLoginFortmatic} />}
```

Comment:
The wallet selector is being opened in the desktop client now to show both wallet connect and wallet link. In this case, I'm not showing fortmatic as it was not being used before in the client and I don't want to added without approval.